JSPatch/JPEngine.m

```diff
@@ -264,17 +272,22 @@ + (JSValue *)_evaluateScript:(NSString *)script withSourceURL:(NSURL *)resourceU
         _regex = [NSRegularExpression regularExpressionWithPattern:_regexStr options:0 error:nil];
     }
     NSString *formatedScript = [NSString stringWithFormat:@";(function(){try{%@}catch(e){_OC_catch(e.message, e.stack)}})();", [_regex stringByReplacingMatchesInString:script options:0 range:NSMakeRange(0, script.length) withTemplate:_replaceStr]];
+    
+    JSValue *ret = nil;
     @try {
         if ([_context respondsToSelector:@selector(evaluateScript:withSourceURL:)]) {
-            return [_context evaluateScript:formatedScript withSourceURL:resourceURL];
```

Comment:
咦，看来我脑残了，我是用改过之后，try里面没有return试的，跟代码原来的写法不一样，所以得出了错误的结论

Dake

> 在 2016年6月2日，11:39，bang <notifications@github.com> 写道：
> 
> In JSPatch/JPEngine.m:
> 
> >      @try {
> >          if ([_context respondsToSelector:@selector(evaluateScript:withSourceURL:)]) {
> > -            return [_context evaluateScript:formatedScript withSourceURL:resourceURL];
> 。。显然你没试过
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>